packages/fixtures/boards/usbvil/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "skipSnapshot": true,
```

Comment:
Feel free to remove this line from any of the boards! It's in `8-bit mixtape` because the complicated silkscreens makes it take _forever_ to snapshot. I remember bus-pirate may be a little long, too, but feel free to use your best judgement